xl-op/blueprint.yaml

```
@@ -558,32 +531,32 @@ spec:
       saveInXlvals: true
       ignoreIfSkipped: true
       overrideDefault: true
-      promptIf: !expr "ServerType != 'dai-release-runner' && (ProcessType == 'install' || ProcessType == 'upgrade')"
+      promptIf: !expr "ServerType != 'dai-release-remote-runner' && (ProcessType == 'install' || ProcessType == 'upgrade')"
       prompt: "Enter the operator image tag (eg: <tagName> from <repositoryName>/<imageName>:<tagName>):"
-      default: 23.3.0-beta.6
-      description: Enter the operator image tag to use
+      default: 23.3.0-917.1722
+      description: Enter the operator image tag to use, for example 23.3.5
```

If this is possible, we can avoid hardcoded version in description field. Otherwise we can leave it.
```suggestion
      description: Enter the operator image tag to use, for example !value: OperatorImageTag.default
```